src/main/plugin/iso19139.ca.HNAP/templates/hnap_nonspatial-english.xml

```diff
@@ -320,13 +338,13 @@
                   <gco:CharacterString/>
                   <gmd:PT_FreeText>
                      <gmd:textGroup>
-                        <gmd:LocalisedCharacterString locale="#fra"/>
+                        <gmd:LocalisedCharacterString locale=""/>
```

Comment:
Why are you removing the locale code?  This line is not required if locale code is removed

Reply:
I just did a load and save, perhaps that process is removing the code?